### ISSUE TITLE
[Bug] Mount Bars Overwritten

### ISSUE THREAD

**Issue by Diorik** [zacharied/FFXIV-Plugin-HudManager#38]
Mount action bars such as Rolling Tankard are having the primary hotbar overwritten. I believe there used to be a config value that controlled this but I no longer see it in the UI. Unsure if the other bar overwrite states are affected.

**Comment by Diorik** [comment]
The special mounts from beast tribes don't seem to be affected.

**Comment by zacharied** [comment]
Wow, that's a pain. I only tested on beast tribe mounts because that's all I can access at the moment.

Quick clarifying question, when you say the beast tribe mounts aren't affected, do you mean the quest-specific mount that you talk to an NPC to access? Or the regular version of the mount? or both? Just a bit strange since Rolling Tankard is technically also a beast tribe mount.

**Comment by Diorik** [comment]
Sorry, the quest mounts are not broke, but the vendored player version of tankard is.

**Comment by Diorik** [comment]
It looks like quest mounts are now also broken after the most recent update - I didn't try to repro so it could have been a one off. Will check next time the hippo daily rolls around.

**Comment by Diorik** [comment]
Strange new behavior since patch, now my hotbars are being blanked out after **exiting** a quest mount. Looks like some condition/flag is staying true after the exit and anything that counts for a HUD update prompts the manager to try and swap to an empty bar (where I assume the mount buttons were?).

**Comment by zacharied** [comment]
Hopefully

**Comment by nebel** [comment]
Since there's been no reports of this happening after #72, I'll go ahead and assume this is fixed. However the fix is kind of a hack so there's a possibility it make happen again in the future. If so, please open a new issue.
